locale/fi/home.json

```diff
@@ -2,7 +2,7 @@
   "title": "Etusivu",
   "intro.title": "Suomi.fi Design System – digipalvelujen yhteiset rakennuspalikat",
   "intro.description": "Suomi.fi Design System tarjoaa kokoelman uudelleenkäytettäviä ja dokumentoituja käyttöliittymäkomponentteja. Kaikki komponentit on toteutettu ja testattu saavutettavuusvaatimusten mukaisesti. ",
-  "note.title": "Tämä on Suomi.fi Design Systemin julkinen kehitysversio (beta)",
+  "note.title": "Tämä on Suomi.fi Design Systemin julkinen kehitysversio",
   "note.description": "Osallistu uuden palvelun kehittämiseen! Anna palautetta ja liity keskusteluun Suomidigin kehittäjät #designsystem Slack- kanavalla.",
   "note.link.text": "Liity Slack -kanavalle",
   "note.link.url": "https://suomidigi-kehittajat.slack.com/",
```

Comment:
Wasn't there discussion that there no longer should be mention to forward to Suomidigi Slack?

Reply:
That seems to be an old relic, since it wasn't touched in this one and it isn't on the page either. I'll try to spot and remove the unused locale entries.